core/modules/system/system.module

```diff
@@ -168,8 +168,20 @@ function system_permission() {
     'administer software updates' => array(
       'title' => t('Administer software updates'),
       'restrict access' => TRUE,
-      'warning' => t('Run site updates for Backdrop projects.'),
+      'warning' => t('Allows running site updates (update.php) for Backdrop projects.'),
     ),
+  );
+  // If backups are not configured, do not show related permission.
+  if (settings_get('backups_directory')) {
+    $permissions += array(
```

Comment:
That looks like a typo: `'backupS_directory'` with "s" - as a result, the permissions do no show, currently.

Reply:
Thanks, fixed!